File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.py
def solution(num_list):
    answer = [0,0]
    even = 0
    odd = 0
    
    for i in range(len(num_list)):
        if num_list[i] % 2 == 0:
            even += 1
        else:
            odd += 1
            
    answer[0] = even
    answer[1] = odd
    
    return answer

File: 프로그래머스/0/120824. 짝수 홀수 개수/짝수 홀수 개수.js
function solution(num_list) {
    var answer = [];
    var even = 0;
    var odd = 0;

    for(let x of num_list) {
        if(x%2==0) {
            even += 1;
        } else {
            odd += 1
        }
    }
    
    answer.push(even, odd)
    return answer;
}